modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -23,7 +23,7 @@ GA:: General Availability
 
```

Comment:
`1.11.x` is TP. 
cc: @vdemeester @concaf @ppitonak @PuneetPunamiya

Reply:
yes

Reply:
Incorporated this change. Thanks